packages/theme-toolkit/src/component-stories-utrecht.tsx

```diff
@@ -2026,7 +2026,27 @@ export const UTRECHT_COMPONENT_STORIES: ComponentStory[] = [
       </Link>
     ),
     detectTokens: {
-      anyOf: ['utrecht.link.color', 'utrecht.link.text-decoration', 'utrecht.link.text-decoration-thickness'],
+      anyOf: [
+        'utrecht.link.color',
+        'utrecht.link.text-decoration',
+        'utrecht.link.text-decoration-thickness',
+        'utrecht.link.underline-offset',
```

Comment:
deze token bestaat niet

Reply:
Oh ik zie het al het moest text.underline-offset zijn of kijk ik ergens verkeerd?

<img width="250" alt="Screenshot 2024-12-30 at 10 32 02" src="https://github.com/user-attachments/assets/b09172da-1521-49b9-b0be-3309b1428587" />